draft-ietf-tls-sni-encryption.md

```diff
@@ -39,13 +39,12 @@
 
 .# Abstract
 
-This draft describes the general problem of encryption of
-the Server Name Identification (SNI) parameter. The proposed
-solutions hide a Hidden Service behind a Fronting Service,
-only disclosing the SNI of the Fronting Service to external
+This draft describes the general problem of encrypting the
+Server Name Identification (SNI) TLS extension. The proposed
```

Comment:
Looking at this again, "parameter" is defensible, since we do not need to encrypt literally the Extension as-is, but rather its semantics conents.

Reply:
Works for me. I'll replace "extension" with "parameter."